app/2.0/docs/tools/glossary.md

```diff
@@ -0,0 +1,162 @@
+---
+title: "Polymer Tools Glossary"
+---
+
+<!-- toc -->
+
+# A
+## app shell
+In a progressive web app, the app shell contains the minimum CSS, HTML and JavaScript to load the basic UI. [Polymer CLI](#polymer-cli) is designed to work with apps that use an app shell architecture. See [Instant Loading Web Apps with an Application Shell Architecture](https://developers.google.com/web/updates/2015/11/app-shell) for more information.
+
+# B
+## bower
+bower is a package manager for the Web. The Polymer toolchain uses bower to manage and install elements for reuse in apps and in other elements. See [bower.io](https://bower.io) for more information.
+
+## bower.json
+`bower.json` is a manifest used by bower for configuring packages for use as dependencies. See [the `bower.json` specification](https://github.com/bower/spec/blob/master/json.md) for more information.
+
+# C
+## crisper
+crisper extracts inline scripts from an HTML file. This is useful if you need to run a transformation on your JavaScript, such as a minifier or a transpiler. You could also use crisper to separate JavaScript from HTML in order to deploy in an environment that uses CSP (Content Security Policy).
+
+# D
+## dependency
+In the context of Polymer apps, a dependency is a resource (a file or set of files) required by your application, such as HTML files or JavaScript libraries. Dependencies should be declared in [`polymer.json`](#polymer-json) to ensure they are included in your build. See also: [dynamic dependency](#dynamic-dependency), [static dependency](#static-dependency), [transitive dependency](#transitive-dependency).
+
+In the context of the Polymer data system, a dependency is a piece of data that an observer, computed property or computed binding depends on.
+
+## dependency variant
```

Comment:
- `dependency variants` or just `variants` would be a more accurate title, since we think about these in terms of adding collections of variants. 
- variants are useful for element projects only at the moment. They are useful for testing elements against Polymer 1.x & Polymer 2.x (where bower.json would normally only let you install and test against one of these). I would change any references to your "app" here to reference your "element" instead.